generator/generate.ts

```diff
@@ -139,7 +139,6 @@ async function generateSchema(readme: string, tmpFolder: string) {
         `--output-folder=${tmpFolder}`,
         `--multiapi`,
         '--title=none',
-        '--pass-thru:subset-reducer',
```

Comment:
Is it safe to remove this? It was added as we had some issues with automatic generation of certain RPs.

Reply:
Ah, this was fixed by https://github.com/Azure/autorest.azureresourceschema/issues/52? If so, feel free to resolve this. 

Please could you test `npm run generate-single -- resources/resource-manager` and make sure it still works after this change?

Reply:
This parameter turn out to be still needed. I'll recreate the PR.